resources/helm/v2.1/gateways/istio-ingress/templates/meshexpansion.yaml

```diff
@@ -1,81 +0,0 @@
-{{ $gateway := index .Values "gateways" "istio-ingressgateway" }}
```

Comment:
We need to determine if we need to modify anything in SMCP for this change.  We have some fields for mesh expansion, but they're not supported or tested, so maybe we just document them as being deprecated.

Reply:
@luksa where can we do that?